code/web/src/modules/StyleSurvey/StyleSurvey.js

```diff
@@ -94,79 +97,103 @@ class StyleSurvey extends Component {
 					category: 'Tops',
 					images: [
 						{
+							id: 'athletic-top',
```

Comment:
Having an `id` property for each image makes things a lot less complicated and in my opinion, targeting its sibling elements became more straightforward.